src/nr_openai_observability/langchain_callback.py

```diff
@@ -250,3 +258,6 @@ def _get_model(self, serialized: Dict[str, Any], **kwargs: Any) -> str:
             model = invocation_params.get("_type", "")
 
         return model
+
+    def _save_metadata(self, metadata):
+        set_conversation_id(metadata.get("conversation_id", None))
```

Comment:
Does it make sense for this `conversation_id` to come through the metadata to the callback vs the `set_conversation_id` context var helper?

Reply:
I'll spend some time double checking this morning, but I wasn't able to see this metadata in any calls to bedrock or openAI where we actually use this conversation ID, only during the LangChain calls -